relays/client-substrate/src/client.rs

```diff
@@ -435,18 +435,25 @@ impl<C: Chain> Client<C> {
 	/// Note: The given transaction needs to be SCALE encoded beforehand.
 	pub async fn submit_signed_extrinsic(
 		&self,
-		extrinsic_signer: C::AccountId,
-		signing_data: SignParam<C>,
```

Comment:
:+1:

Reply:
We shall probably do the same with the `submit_and_watch_signed_extrinsic` (not a part of this PR, of course).

Reply:
I did these changes for `submit_and_watch_signed_extrinsic` also. Also added a function for creating the signing params in order to avoid duplicating this logic.